js/views/modals/Settings/SettingsPage.js

```diff
@@ -0,0 +1,72 @@
+import loadTemplate from '../../../utils/loadTemplate';
+import { View } from 'backbone';
+import select2 from 'select2'; // eslint-disable-line no-unused-vars
+
+export default class extends View {
+  constructor(options = {}) {
+    super({
+      className: 'settingsGeneral',
+      events: {
+      },
+      ...options,
+    });
+
+    // temp data. This view will need the user model, the languages, the countries, and the
+    // currencies.
+
+    this.profileModel = {
+      handle: '@example',
+      name: 'exampleName',
+      location: 'UNITED_STATES',
+      about: 'example about',
+      shortDescription: 'example short description',
+      website: 'http://example.website.com',
+      email: 'example@email.com',
+      nsfw: false,
+      vendor: false,
+      moderator: false,
+      primaryColor: '',
+      secondaryColor: '',
+      textColor: '',
+      followerCount: 0,
+      followingCount: 0,
+      listingCount: 0,
+    };
+
+    this.countryList = [
+      { code: 'USA', dataName: 'UNITED_STATES', name: 'United States' },
+      { code: 'DZD', dataName: 'ALGERIA', name: 'Algeria' },
+    ];
+
+    this.options = {
+      // test data
+      countryList: this.countryList,
+    };
+
+    this.options = Object.assign({}, options, this.profileModel, this.options);
```

Comment:
It certainly doesn't need to be on options. Originally I was going to pass the data in as an option from the Settings parent, but didn't fix it after changing my mind.

In any case, it's just fake data until the models are wired into it.